ms_game_chat/backend/requirements.txt

```diff
@@ -3,4 +3,4 @@ psycopg[binary]
 channels~=4.0
 daphne~=4.0
 channels_redis~=4.1
-ft_jwt==0.1.0
\ No newline at end of file
+ft_jwt==0.1.2
```

Comment:
@mariekart42  update to 1.0.2 improves how the time is calculated. before i was using deprecated methods